projects/core/utils/dom/process-icon.ts

```diff
@@ -7,8 +7,8 @@ export function processIcon(source: string, name: string): string {
         return source;
     }
 
-    const src = source.substring(source.indexOf(START));
-    const attributes = src.substring(0, src.indexOf('>'));
+    const src = source.slice(Math.max(0, source.indexOf(START)));
+    const attributes = src.slice(0, Math.max(0, src.indexOf('>')));
```

Comment:
`substring` doesn't seem deprecated and it doesn't require extra `Math.max`, are u sure about replacing?